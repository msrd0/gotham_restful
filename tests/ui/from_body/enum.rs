#[macro_use]
extern crate gotham_restful;

#[derive(FromBody)]
enum FromBodyEnum {
	SomeVariant(Vec<u8>),
	OtherVariant(String)
}

fn main() {}
