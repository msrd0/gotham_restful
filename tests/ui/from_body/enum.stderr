error: #[derive(FromBody)] only works for structs
 --> $DIR/enum.rs:5:1
  |
5 | enum FromBodyEnum {
  | ^^^^
